core/types/vote.go

```diff
@@ -0,0 +1,85 @@
+package types
+
+import (
+	"sync/atomic"
+
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/crypto/bls"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	BLSPublicKeyLength = 48
+	BLSSignatureLength = 96
+)
+
+type BLSPublicKey [BLSPublicKeyLength]byte
+type BLSSignature [BLSSignatureLength]byte
+type ValidatorsBitSet uint64
+
+type VoteData struct {
+	SourceNumber uint64
```

Comment:
please help to add more comments to the new defined structs. not only this one.

Reply:
fixed